src/server/MapPlaylist.ts

```diff
@@ -90,6 +90,7 @@ export class MapPlaylist {
           Iceland: 2,
           SouthAmerica: 1,
           KnownWorld: 2,
+          DeglaciatedAntarctica: 2,
```

Comment:
I'm not sure if this should be showing up more often than South America.
```suggestion
          DeglaciatedAntarctica: 1,
```

Reply:
I think new maps should have a chance of being played by the players. I don't think the South America map is very liked